_posts/en/newsletters/2020-06-10-newsletter.md

```diff
@@ -0,0 +1,329 @@
+---
+title: 'Bitcoin Optech Newsletter #101'
+permalink: /en/newsletters/2020/06/10/
+name: 2020-06-10-newsletter
+slug: 2020-06-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a blog post and paper about using
+eclipse attacks against Bitcoin software in order to steal from LN
+channels and describes the history and impact of a fee overpayment
+attack that affects hardware wallets.  Also included are our regular
+sections with the summary of a Bitcoin Core PR Review Club meeting,
+descriptions of recent software releases, and summaries of recent
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Check hardware wallet compatibility:** if your software or your
+  processes allow using Trezor devices to spend segwit inputs, check
+  whether your system remains compatible with [Trezor's latest firmware
+  update][trezor update].  Other hardware wallets may release similar
+  updates in the future; contact the manufacturers for more information
+  about their plans.  See the *news* item below about the fee
+  overpayment attack for the motivation for this
+  change.
+
+## News
+
+- **Time dilation attacks against LN:** Gleb Naumenko and Antoine Riard
+  [posted][naumenko td] to the Bitcoin-Dev mailing list a summary of a
+  [blog post][td blog] they wrote that itself summarizes a [paper][td
+  paper] they authored about using [eclipse attacks][topic eclipse attacks]
+  to steal money from LN channels.  This extends the analysis by Riard
+  described in [Newsletter #77][news77 eclipse].  In short, an attacker
+  gains control over all of an LN node's connections to the Bitcoin P2P
+  network and delays the relay of new block announcements to the victim.
+  After the victim's view of the block chain is far enough behind the
+  public consensus view, the attacker closes their channels with the
+  victim in an outdated state that pays the attacker more than the
+  latest state would.  The eclipse attack is used to prevent the victim
+  from seeing the close transactions until after the dispute period is
+  over and the attacker has withdrawn their illicit gains to an address
+  entirely under the attacker's control.
+
+    The paper describes an attack that can steal from lightweight
+    clients in as little as two hours.  For attacks against LN nodes
+    backed by a local Bitcoin full verification node, their shortest
+    duration attack requires eclipsing the victim for at least 24 hours.
+    The authors note that the attacks probably apply to other
+    time-sensitive contract protocols and that the best solutions to the
+    problem all involve improving eclipse attack resistance.
```

Comment:
The AltNet prroposal (https://github.com/bitcoin/bitcoin/pull/18988) has been quite motivated by this research. The current base layer p2p assumptions are limited with regards to LN security model.

Reply:
Blog post invite future research on inter-layer mapping or native-LN attacks. For more context, see IRC logs of LN dev meeting of today, which is about relevance of those topics.